client/src/App.js

```diff
@@ -1,10 +1,67 @@
 import "./App.css";
+import VideoCard from "./component/VideoCard";
+import AddVideo from "./component/AddVideo";
+import React from "react";
+import { useState, useEffect } from "react";
+import Data from "./exampleresponse.json";
 
 function App() {
+  const [videosList, setVideosList] = useState(Data);
+  const [deletedVideoId, setDeletedVideoId] = useState(null);
+  // const [newList, setNewList] = useState({});
+  const [formData, setFormData] = useState({ title: "", url: "" });
+
+
+  useEffect(function removeSelectedVideo() {
+    // let targetedVideoIndex = videosList.findIndex((video) =>video.id == deletedVideoId);
+    let filteredVideos = videosList.filter((video) => video.id != deletedVideoId);
```

Comment:
What does your linter say about this line when you run your app locally?

How can linters help you improve your code?

Reply:
Thanks for pointing out this Issue Sally , It was my question too
I wanted to use !==  instead of != but when I use double equal sign the code doesn't work , I wanted to ask the reason because I can't figure the reason  out